lib/cocoapods/validator.rb

```diff
@@ -608,8 +608,15 @@ def parse_xcodebuild_output(output)
     #         returns its output (both STDOUT and STDERR).
     #
     def xcodebuild
-      command = 'xcodebuild clean build -target Pods'
-      command << ' CODE_SIGN_IDENTITY=- -sdk iphonesimulator' if consumer.platform_name == :ios
+      command = 'xcodebuild clean build -target Pods
```

Comment:
I feel a bit stupid now ;-) Actually the `'` was on the next line, but still this makes all tests fail…